### DISCUSSION TITLE
1.0.6 nicht durch HACS installierbar?

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#398] [Q&A] [answered]
Ich wollte mal die BATTERY_STATE Integration, die mit der 1.0.6 Einzug hielt, testen.
Komischerweise meint HACS auf beiden Test HA's die 1.0.5 sei die aktuellste Version.

Wurde da irgendwas geändert?
![HACS_no_106](https://user-images.githubusercontent.com/54898249/162568119-8baf4176-8f31-4d25-b175-bc0396e8fd42.jpg)

**Comment by SukramJ** [comment, marked as answer]
Nein, 1.0.6 wurde auch noch nicht veröffentlicht.

**Reply by Baxxy13** [reply]
Ah, na dann ist das klar. Danke.
